packages/components/organisms/f-takeawaypay-activation/src/tenants/en-AU.js

```diff
@@ -0,0 +1,26 @@
+const messages = {
+    locale: 'en-AU',
+
+    messages: {
+        titleActivationAvailable: 'Welcome back, {consumerGivenName}. Let\'s activate Menulog Pay on your account.',
+        titleActivationNotLoggedIn: 'Let\'s get you started with Menulog Pay.',
+        titleActivationSuccessful: 'All systems go.',
+        titleActivationFailed: ' We can\'t activate Menulog Pay on your account right now, sorry.',
+        descriptionActivationAvailable1: 'Menulog Pay will be activated for',
+        descriptionActivationAvailable2: 'Not you? Choose a different account or sign up.',
+        descriptionActivationNotLoggedIn: 'Do you already have a Menulog account?',
+        descriptionActivationFailed: 'Try again or get in touch with your Menulog Pay administrator for help.',
+        descriptionActivationSuccessful: 'Menulog Pay is now activated on your account.'
```

Comment:
Can you use unicode apostrophe characters here, to avoid escaping, and also to function better semantically?
```suggestion
        titleActivationAvailable: 'Welcome back, {consumerGivenName}. Let’s activate Menulog Pay on your account.',
        titleActivationNotLoggedIn: 'Let’s get you started with Menulog Pay.',
        titleActivationSuccessful: 'All systems go.',
        titleActivationFailed: ' We can’t activate Menulog Pay on your account right now, sorry.',
        descriptionActivationAvailable1: 'Menulog Pay will be activated for',
        descriptionActivationAvailable2: 'Not you? Choose a different account or sign up.',
        descriptionActivationNotLoggedIn: 'Do you already have a Menulog account?',
        descriptionActivationFailed: 'Try again or get in touch with your Menulog Pay administrator for help.',
        descriptionActivationSuccessful: 'Menulog Pay is now activated on your account.'
```